src/EncompassRest/LoanPipeline/LoanPipelineCursor.cs

```diff
@@ -23,59 +23,53 @@ internal LoanPipelineCursor(EncompassRestClient client, string cursorId, int cou
+        public Task<LoanPipelineData> GetItemAsync(int index) => GetItemAsync(index, null, CancellationToken.None);
 
-        public async Task<LoanPipelineData> GetItemAsync(int index, CancellationToken cancellationToken)
+        public Task<LoanPipelineData> GetItemAsync(int index, IEnumerable<string> itemFields) => GetItemAsync(index, itemFields, CancellationToken.None);
+
+        public async Task<LoanPipelineData> GetItemAsync(int index,IEnumerable<string> itemFields, CancellationToken cancellationToken)
         {
             Preconditions.GreaterThanOrEquals(index, nameof(index), 0);
             Preconditions.LessThan(index, nameof(index), Count, nameof(Count));
 
-            var data = await GetItemsInternalAsync(index, 1, cancellationToken).ConfigureAwait(false);
+            var data = await GetItemsInternalAsync(index, 1,itemFields, cancellationToken).ConfigureAwait(false);
             return data[0];
         }
 
-        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count) => GetItemsAsync(start, count, CancellationToken.None);
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count) => GetItemsAsync(start, count,null, CancellationToken.None);
+
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count, IEnumerable<string> itemFields) => GetItemsAsync(start, count, itemFields, CancellationToken.None);
 
-        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count, CancellationToken cancellationToken)
+        public Task<List<LoanPipelineData>> GetItemsAsync(int start, int count,IEnumerable<string> itemFields, CancellationToken cancellationToken)
         {
             Preconditions.GreaterThanOrEquals(start, nameof(start), 0);
             Preconditions.LessThan(start, nameof(start), Count, nameof(Count));
             Preconditions.GreaterThan(count, nameof(count), 0);
-            Preconditions.LessThanOrEquals(start + count, $"{nameof(start)} + {nameof(count)}", Count, nameof(Count));
+            //Preconditions.LessThanOrEquals(start, nameof(start) , Count, nameof(Count));
+            
 
-            return GetItemsInternalAsync(start, count, cancellationToken);
+            return GetItemsInternalAsync(start, count,itemFields, cancellationToken);
         }
 
-        private async Task<List<LoanPipelineData>> GetItemsInternalAsync(int start, int count, CancellationToken cancellationToken)
+        private async Task<List<LoanPipelineData>> GetItemsInternalAsync(int start, int count, IEnumerable<string> itemFields, CancellationToken cancellationToken)
         {
-            List<LoanPipelineData> data = null;
-            do
+            var retrievedData = await Client.Pipeline.ViewPipelineCursorInternalAsync(CursorId, null, itemFields == null?Fields:itemFields, start, count, cancellationToken, nameof(GetItemAsync), async response =>
```

Comment:
use null coalesce operator instead of the ternary operator. e.g. `fields ?? Fields`